keyboards/phantagom/baragon/info.json

```diff
@@ -0,0 +1,75 @@
+{
+    "manufacturer": "Dennis Kruyt",
+    "keyboard_name": "phantagom/baragon",
+    "maintainer": "dkruyt",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true
+    },
+    "rgblight": {
+        "led_count": 12,
+        "pin": "GP15",
+        "hue_steps": 10,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 0]
+    },
```

Comment:
`0, 0` is the default value, no need to duplicate its configuration. And since bootmagic is enabled on the `features` block this whole section can be removed
```suggestion
```